src/main/scala/lms/thirdparty/cuda/cudnn.scala

```diff
@@ -35,6 +35,94 @@ object CUDNNTypeLess extends Dsl with CLibs {
     assert(result.t == manifest[CUDNN_RESULT], "CUDNN_CHECK must take the CUDNN_RESULT type as input")
     UNIT(LIB_FUNCTION(manifest[Unit], "CUDNNCHECK", result.x)(Seq[Int](), Seq[Int](), Set[Int](), Adapter.CTRL))
   }
+
+  class CUDNN_TENSOR_DESCRIPTOR(override val x: Backend.Exp) extends TOP(x)
+  class CUDNN_FILTER_DESCRIPTOR(override val x: Backend.Exp) extends TOP(x)
+  class CUDNN_CONV_DESCRIPTOR(override val x: Backend.Exp) extends TOP(x)
```

Comment:
since I just merged my last PR, it is likely these classes will fail compilation.

I think all needed is to add override val useOldMetadata: Boolean in the parameters.

Reply:
fixed in 3d6e42c